src/js/components/award/AgencyInfo.jsx

```diff
@@ -0,0 +1,87 @@
+/**
+ * AgencyInfo.jsx
+ * Created by Emily Gullo 01/20/2017
+ **/
+
+import React from 'react';
+
+const propTypes = {
+    toggleAgency: React.PropTypes.func,
+    agencySet: React.PropTypes.object,
+    agencyTitle: React.PropTypes.string,
+    setAwardValues: React.PropTypes.func,
+    toptierAgency: React.PropTypes.object,
+    subtierAgency: React.PropTypes.object,
+    officeAgency: React.PropTypes.object
+};
+
+export default class AgencyInfo extends React.Component {
```

Comment:
Consider moving the state to track the dropdown agency type here instead of the parent container.